desktop/apps/categories3/helpers/utils.js

```diff
@@ -0,0 +1,3 @@
+import _ from 'underscore'
+
+export const alphabetizeGenes = genes => _.sortBy(genes, gene => gene.name)
```

Comment:
This is just a thought, but thinking -- maybe instead of `helpers/utils` it could be `utils/alphabetizeGenes.js`? It's always nice when one can take a glance at the file-system and immediately see whats going on, especially for things that have definite bounds and not too many files.

Reply:
Yeah, I like that.